.github/workflows/test.yml

```
@@ -6,6 +6,30 @@ on:
   pull_request:
 
 jobs:
+  msrv-check:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v2
+
+      - name: Get MSRV
+        run: |
+          RUST_MSRV="$(cat cargo-near/Cargo.toml | sed -n 's/rust-version *= *"\(.*\)"/\1/p')"
```

Cut some corners here for cross-platform compatibility, seems fine.. Or should we do this?

```suggestion
          RUST_MSRV="$(cat cargo-near/Cargo.toml | sed -n 's/rust-version[[:space:]]*=[[:space:]]*"\(.*\)"/\1/p')"
```

---

if it works on CI, I don't personally see a downside

---

> Edit: are you thinking about windows compatibility?

Oh, no.. I haven't tested that. It was macOS I was thinking of. I used `\s` first, and it didn't support that for some reason, so I used a hard space.